app/views/govuk_publishing_components/components/_document_list.html.erb

```diff
@@ -72,6 +71,37 @@
         <% if item[:subtext] %>
           <p class="gem-c-document-list__subtext"><%= item[:subtext] %></p>
         <% end %>
+        
+        <% if item.include?(:parts) && item[:parts].any? %>
+          <ul class="gem-c-document-list__children govuk-list">
+            <% item[:parts].each do |part| %>
+              <li class="gem-c-document-list-child">
+                <%=
+                  item_classes = "gem-c-document-list-child__heading #{brand_helper.color_class}"
```

Comment:
Not sure if this benefits from being a separate variable, especially as we're now adding another class separately on line 87 too. We should either consistently store all classes in `item_classes` or not at all?

Reply:
Updated.